src/containerapp/azext_containerapp/_sdk_models.py

```diff
@@ -7337,44 +7337,44 @@ def __init__(
 class JobScale(_serialization.Model):
     """Container Apps Job scaling configurations.
 
-    :ivar min_replicas: Optional. Minimum number of container replicas.
-    :vartype min_replicas: int
-    :ivar max_replicas: Optional. Maximum number of container replicas. Defaults to 10 if not set.
-    :vartype max_replicas: int
+    :ivar min_executions: Optional. Minimum number of container replicas.
+    :vartype min_executions: int
+    :ivar max_executions: Optional. Maximum number of container replicas. Defaults to 10 if not set.
```

Comment:
The description please consistent with [azure-rest-api-specs/specification/app/resource-manager/Microsoft.App/stable/2023-05-01/Jobs.json at 11bbc2b1df2e915a2227a6a1a48a27b9e67c3311 · Azure/azure-rest-api-specs (github.com)](https://github.com/Azure/azure-rest-api-specs/blob/11bbc2b1df2e915a2227a6a1a48a27b9e67c3311/specification/app/resource-manager/Microsoft.App/stable/2023-05-01/Jobs.json#L732)